trackintel/preprocessing/staypoints.py

```diff
@@ -35,8 +35,7 @@ def generate_locations(
         - 'dbscan' : Uses the DBSCAN algorithm to cluster staypoints.
 
     epsilon : float, default 100
-        The epsilon for the 'dbscan' method. if 'distance_metric' is 'haversine'
-        or 'euclidean', the unit is in meters.
+        The epsilon for the 'dbscan' method. For any "distance_metric" the unit is meters.
```

Comment:
Hm, yes one could input any distance metric. But on the other hand we only implement haversine and euclidean because we are using either our functions or the geopandas ones. So the distance_metric gets ignored past the dbscan.

Reply:
Mhm okay I see your point, but it seems a bit weird to me to write in the docstring 
`Any mentioned below are possible:        https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances.html"`, and then effectively only using/allowing haversine or euclidean. But it's not a big deal; it could maybe be explained a bit more.

Reply:
I removed it as my changes were false, as we somewhat support other distance metrics for which the metric is not in meters.
But if you think that this docstring is still an issue, I'll open a new issue for that.